### PR TITLE
Added burger without integration with header and fixed path of images

### PR DESCRIPTION
**Pull request by DimaKobzar7**

### CODE REVIEW

File: fe/src/components/BurgerMenu/BurgerMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+export const BurgerMenu: React.FC = () => {
+  const [showBurgerMenu, setShowBurgerMenu] = useState(false);
+
+  const changeBurgerState = () => {
+    setShowBurgerMenu(true);
+  };
+
+  return (
+    <header
```
Comment (VolodymyrChuyko) [inline fe/src/components/BurgerMenu/BurgerMenu.tsx:22]:
```suggestion
    <div
```
the header tag should be used only for its intended purpose

File: fe/src/components/BurgerMenu/BurgerMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          className={({ isActive }) =>
+            classNames('burger-menu__btn', {
+              'is-active': isActive,
+            })
+          }
+        >
+          <img src={bag} alt="bag" />
+        </NavLink>
+      </div>
+    </header>
```
Comment (VolodymyrChuyko) [inline fe/src/components/BurgerMenu/BurgerMenu.tsx:79]:
```suggestion
    </div>
```

File: fe/src/components/BurgerMenu/BurgerMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  const [showBurgerMenu, setShowBurgerMenu] = useState(false);
+
+  const changeBurgerState = () => {
+    setShowBurgerMenu(true);
+  };
+
+  return (
+    <header
+      className={classNames('burger-menu', {
+        'hide-burger-menu': showBurgerMenu,
```
Comment (VolodymyrChuyko) [inline fe/src/components/BurgerMenu/BurgerMenu.tsx:24]:
```suggestion
        'hide-burger-menu': hideBurgerMenu,
```
Doesn't look logical. Could you change the name of the variable or class so they fit each other

File: fe/src/components/BurgerMenu/BurgerMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      })}
+    >
+      <div className="burger-menu__wrap">
+        <div className="burger-menu__logo">
+          <a href="#">
+            <img src={logo} alt="nice-gadgets" />
+          </a>
+        </div>
```
Comment (VolodymyrChuyko) [inline fe/src/components/BurgerMenu/BurgerMenu.tsx:32]: can you use the Logo component?

**Review by VolodymyrChuyko — CHANGES_REQUESTED** [review]
Good job. Just small issues

File: fe/src/components/BurgerMenu/BurgerMenu.tsx (hunk 1 above)
Comment (DimaKobzar7) [reply fe/src/components/BurgerMenu/BurgerMenu.tsx:22]: I used it because it is header. Another part of it

**Pull request closed without merging**